crates/sui-replay-2/src/execution.rs

```diff
@@ -47,6 +48,17 @@ pub struct ReplayExecutor {
     metrics: Arc<LimitsMetrics>,
 }
 
+// Returned struct from execution. Contains all the data related to a transaction.
+// Transaction data and effects (both expected and actual) and the caches containing 
+// the objects used during execution.
+pub struct TxnContextAndEffects {
```

Comment:
btw this is a pretty ludicrous name but cannot come up with anything intelligent.
Feel free to suggest alternatives